app/helpers/decidim/solutions/application_helper.rb

```diff
@@ -6,6 +6,7 @@ module Solutions
     #
     module ApplicationHelper
       include Decidim::RichTextEditorHelper
+      include Decidim::PaginateHelper
```

Comment:
can't we add it to the controllers that really need it as a helper instead of in the ApplicationHelper?